lib/HTMLElement.js

```diff
@@ -19,4 +19,22 @@ module.exports = class HTMLElement extends Element {
   get contentEditable() {
     return this.hasAttribute("contenteditable") || false;
   }
+
+  get draggable() {
+    return !!this.getAttribute("draggable");
+  }
+
+  set draggable(value) {
+    this.getAttribute("draggable", !!value);
+    return value;
+  }
+
+  get droppable() {
```

Comment:
Maybe I'm missing something but as far as I can find there is no droppable attribute in the HTML spec?

Reply:
I'll get it removed, I was wrong